wandb/sdk/launch/launch.py

```diff
@@ -150,22 +151,22 @@ def run(
     """
     if docker_args is None:
         docker_args = {}
-    if _is_wandb_local_uri(api.settings("base_url")):
-        if sys.platform == "win32":
-            docker_args["net"] = "host"
-        else:
-            docker_args["network"] = "host"
-        if sys.platform == "linux" or sys.platform == "linux2":
-            docker_args["add-host"] = "host.docker.internal:host-gateway"
+    # if _is_wandb_local_uri(api.settings("base_url")):
+    #     if sys.platform == "win32":
```

Comment:
is this supposed to be commented out

Reply:
Ah, needs to be deleted. I wanted to make sure that removing this didn't break tests on circle